### PR TITLE
Update the ghcide upstream to be in haskell/ghcide

### PR DESCRIPTION
**Pull request by alanz**
No other change

### CODE REVIEW

**Comment by pepeiborra** [comment]
We have https://github.com/haskell/ghcide/issues/663 to track all the things that need to be merged to ghcide upstream before hls can switch

**Comment by alanz** [comment]
@pepeiborra Great.

This is intended as an administrative commit, prior to an hls "release" tag for the monthly cadence, of the current hls master.

File: .gitmodules [hunk 1]
```diff
@@ -12,6 +12,7 @@
 	path = ghcide
 	# url = https://github.com/digital-asset/ghcide.git
```
Comment (pepeiborra) [inline .gitmodules:13]:
```suggestion
```

File: .gitmodules [hunk 2]
```diff
@@ -12,6 +12,7 @@
...
 	# url = https://github.com/alanz/ghcide.git
-	url = https://github.com/wz1000/ghcide.git
+	# url = https://github.com/wz1000/ghcide.git
+	url = https://github.com/haskell/ghcide.git
```
Comment (pepeiborra) [inline .gitmodules:16]: what is the impact of this change?

Comment (alanz) [reply .gitmodules:16]: Functionally, nothing. It's a vanity commit.

Comment (pepeiborra) [reply .gitmodules:15]: I see!

**Review by pepeiborra — APPROVED** [review]

**Pull request merged**